From 011de2e85813ff496a85404b30891352555de077 Mon Sep 17 00:00:00 2001
From: Jonas <git@jonas-ott.de>
Date: Fri, 17 Feb 2023 18:26:01 +0100
Subject: [PATCH] cmake: use FindPython3 (#1983)

@jonas-ott Thank you for contributing!

Upstream: https://github.com/lensfun/lensfun/commit/011de2e85813ff496a85404b30891352555de077

[Bernd: partial backport to version 0.3.4]
Signed-off-by: Bernd Kuhls <bernd@kuhls.net>
---
 CMakeLists.txt       |  4 ++--
 apps/CMakeLists.txt  | 55 +++++++++++++++++++++-----------------------
 tests/CMakeLists.txt |  4 ++--
 3 files changed, 30 insertions(+), 33 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5ccb9976d..7bb7a6878 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,5 +1,5 @@
-# require at least cmake 2.8.12
-CMAKE_MINIMUM_REQUIRED(VERSION 2.8.12 FATAL_ERROR )
+# require at least cmake 3.12
+CMAKE_MINIMUM_REQUIRED(VERSION 3.12 FATAL_ERROR)
 
 # path for helper modules
 set(CMAKE_MODULE_PATH "${CMAKE_MODULE_PATH}" "${CMAKE_SOURCE_DIR}/cmake/modules")
